packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -111,20 +169,7 @@ const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
             </SelectIconWrapper>
           </ComboboxInputWrapper>
         </FormControlWrapper>
-        <ComboboxListbox {...getMenuProps()}>
-          {isOpen &&
-            items.map((item, index) => {
-              return (
-                <ComboboxListboxOption
-                  {...getItemProps({item, index})}
-                  highlighted={highlightedIndex === index}
-                  key={uid(item)}
-                >
-                  {optionTemplate ? optionTemplate(item) : item}
-                </ComboboxListboxOption>
-              );
-            })}
-        </ComboboxListbox>
+        <ComboboxListbox {...getMenuProps()}>{isOpen && renderListBox()}</ComboboxListbox>
```

Comment:
Since the listbox doesn't need to remain in the DOM to animate in/out, I wonder if we could move the `isOpen` outside?

```
{isOpen ? <ComboboxListbox .... > : null}
```

Reply:
Sure